Containers/talk-recording/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN set -ex; \
         git \
         wget \
         shadow \
+        pulseaudio \
```

Comment:
is libpulse not enough?

Reply:
Pulseaudio is recommended: https://github.com/nextcloud/spreed/pull/8788/files#diff-3e64353968025a91591a94eb471fd10febcb2b16fb933d84c460f2132029d9bbR39

Reply:
so it requires the bin file? since for launchen the lib was enough

Reply:
then libpulse can be removed